src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -140,11 +141,13 @@
             }
         </style>
         <div class="comments-box">
-            <h3 hidden$="{{_computeHidden(comments)}}">Be the first to comment!</h3>
+            <h3 hidden$="{{_computeHiddenHint(comments)}}">Be the first to comment!</h3>
             <div class="line"></div>
-            <template is="dom-repeat" items="{{comments}}">
+            <template is="dom-repeat" items="{{comments}}" as="item" >
                 <div class="comment">
-                    <div class="comment-avatar"><img src="{{_getAvatar(item)}}" /></div>
+                    <div class="comment-avatar">
+                        <iron-image preload placeholder="/assets/icons/shares/commentAvatar.svg" fade sizing="contain" src="{{item.author.avatar.urls.circle}}"></iron-image>
```

Comment:
fade only works if preload is set: https://elements.polymer-project.org/elements/iron-image